### ISSUE TITLE
Fingerprinting/privacy concerns

### ISSUE THREAD

**Issue by ngzhian** [WebAssembly/relaxed-simd#2]
The SIMD proposal did not introduce any new performance based fingerprinting methods. Relaxed SIMD will introduce instructions that are specifically different across:
- architectures
- supported extension
- processor models (reciprocal approximation)

We should get the conversation started early on this, perhaps include reviewers to examine what is acceptable, and think of ways to maintain the privacy.

**Comment by arunetm** [comment]
related discussion here: https://github.com/WebAssembly/relaxed-simd/issues/11

**Comment by ngzhian** [comment]
@dtig let's reuse this issue for what we discussed earlier today about more detailed fingerprinting issues w.r.t web platform baselines.

**Comment by dtig** [comment]
@ngzhian I've added this as a PR (https://github.com/WebAssembly/relaxed-simd/pull/83) instead because multiple threads are hard to follow in issues, and as an easy way to link for privacy reviews. Let me know if you'd prefer it to be in a different format.
